tests/schemas/builder.py

```diff
@@ -16,7 +20,7 @@
     ZipMsgpackSerializer: "zip+msgpack",
     BaseSerializer: None,  # Not serializable
 }
-_BASE_DATA_CLASSES = (BaseData, BaseSignedData, BaseLocalData)
+_BASE_DATA_CLASSES = (BaseData, BaseSignedData, BaseLocalData, Manifest, LocalManifest)
```

Comment:
Should rename `Manifest` -> `BaseManifest` and `LocalManifest` -> BaseLocalManifest`

Reply:
:+1: